release-notes/opensearch-alerting.release-notes-2.1.0.0.md

```diff
@@ -0,0 +1,9 @@
+## Version 2.1.0.0 2022-07-06
```

Comment:
This may have been pulled into the PR because your dev branch is old. Could you rebase your dev branch with the current `main`?